packages/cli/src/commands/index.ts

```diff
@@ -14,6 +14,7 @@ import test from "./test";
 import trace from "./trace";
 import devContracts from "./dev-contracts";
 import verify from "./verify";
+import abigen from "./abigen";
```

Comment:
not a huge fan of continuing the tablegen, worldgen, etc. naming pattern

I could see us eventually combining all of these features under a `codegen` command, maybe with sub commands like `codegen tables`), but in the meantime, something like "codegen abi" or "generate abi" for the CLI command and "world to abi" for the helper methods?

Reply:
hmm I like this line of thinking, lemme marinate on it

It's interesting that `abigen` and `worldgen` have different inputs but basically the same output

Reply:
The command is now `generate-abi` and the helper is just `systemToAbi`